client/views/actions/away.tpl

```diff
@@ -0,0 +1,7 @@
+<span role="button" class="user {{colorClass from}}" data-name="{{from}}">{{mode}}{{from}}</span>
+{{#if text}}
+went away
+<i class="away-message">({{{parse text}}})</i>
+{{else}}
+came back
+{{/if}}
```

Comment:
I'm not saying we should trim `text`. I'm saying the check to know if there is a text should take this into account. This is because when using The Lounge, sending `/away` [will send a `" "` reason](https://github.com/thelounge/lounge/blob/0d8b58425eff9cc2a9f7468fcc678f8b2e1e7156/src/plugins/inputs/away.js#L7) so anyone else using The Lounge would see `nick is now away ( )`.
In such situation, it should not display the parens.